styles.css

```diff
@@ -1,552 +1,623 @@
+  width: 100%;
+}
+
+.button-outlined {
+  font-family: 'Poppins', sans-serif;
+  outline: inherit;
+  border: 0.1em solid #6070ff;
+  border-radius: 0.75em;
+  color: #396df2;
+  background-color: #fff;
+  align-self: flex-start;
+  margin: 1.5em 0;
+  padding: 1em 1em;
+  font-size: 1.1rem;
+  text-decoration: none;
+}
+
+ul.tools {
+  margin: 0;
+  padding: 0 2.5em;
+  align-self: stretch;
+}
+
+h5 {
+  font-weight: 500;
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+ul.tools li {
+  display: inline;
+}
+
+.tools-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1.5em 0;
+}
+
+.tools-item-detail {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  font-weight: 500;
+  background-color: #f7f7f9;
+  border-radius: 0.6em;
+  margin-bottom: 1em;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+  padding-right: 0.5em;
+  padding-left: 1.5em;
+}
+
+.tools-item-detail p {
+  margin-left: 1em;
+}
+
+.tools-item-detail img {
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+.tools-item {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: min-content;
+  justify-content: center;
+}
+
+.project-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  background-color: #fff;
+  border-radius: 0.6em;
+  margin-top: 7em;
+  padding: 1em;
+}
+
+.project-item * {
+  margin: 0.2em 0;
+}
+
+ul.captions {
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+ul.captions li {
+  display: inline;
+}
+
+.counter {
+  display: inline-block;
+  background-image: url('images/counter.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  width: 1.5em;
+  height: 0.5em;
+}
+
+.text-bold {
+  font-weight: 700;
+}
+
+.text-medium {
+  font-weight: 500;
+}
+
+.text-regular {
+  font-weight: 400;
+}
+
+.text-light-grey {
+  color: #7a869a;
+}
+
+ul.project-language {
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 1.3em;
+}
+
+ul.project-language li {
+  display: inline;
+  background-color: #ebebff;
+  margin-right: 0.5em;
+  padding: 0 1em;
+  border-radius: 0.5em;
+}
+
+.text-white {
+  color: #fff;
+}
+
+.contact-me {
+  background-color: #fff;
+}
+
+.contact-me .content-background {
+  background-image: url('images/contact_me_mobile.svg');
+  background-repeat: no-repeat;
+  background-position: center right;
+  background-size: contain;
+  height: 100%;
+  display: flex;
+  background-color: #6070ff;
+  border-top-left-radius: 5em;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.contact-me .content-background .content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.popup-card .card-title .h1 {
+  margin: 0.4em 0 0;
+}
+
+.action-buttons .button-outlined {
+  font-size: 1rem;
+}
```

Comment:
- [ ] I think you should a little bit work on your buttons, you should make sure your buttons design stick to the original buttons in the figma design. Kindly work on it 🙃 🙃 

Your design

<img width="587" alt="Screenshot 2023-04-11 at 18 39 54" src="https://user-images.githubusercontent.com/74501523/231218231-4c779e6d-02e9-48bd-aa22-b0e1f6549c00.png">

Figma design

<img width="587" alt="Screenshot 2023-04-11 at 18 49 03" src="https://user-images.githubusercontent.com/74501523/231218316-e5d8b9ab-b0ac-4fdd-b8a6-82b7f991b9e2.png">